src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
```suggestion
type Shape = 'triangle' | 'circle' | 'rectangle';
```
створи для Shape enum, а не тип